src/algebra/big_operators.lean

```diff
@@ -648,11 +648,11 @@ lemma mul_sum : b * s.sum f = s.sum (λx, b * f x) :=
 
 lemma sum_mul_boole [decidable_eq α] (s : finset α) (f : α → β) (a : α) :
   s.sum (λ x, (f x * ite (a = x) 1 0)) = ite (a ∈ s) (f a) 0 :=
-by simp
+by simp only [mul_ite, mul_one, mul_zero, sum_ite_eq]
```

Comment:
Is there a need to change to `simp only` here?

Reply:
Yes, there had been, but I think I've solved it again(?), see below.